docs/en/stack/ml/setup.asciidoc

```diff
@@ -92,14 +92,15 @@ For read-only access to the {ml-features} in {kib}, you must have:
 * [ ] `read` index privileges on destination indices (for {dfanalytics-jobs}
 only)
 
+[[upload-file-security-privileges]]
 To upload files in {kib} with the *{file-data-viz}*, you must have:
 
 [%interactive]
-* [ ] `all` {kib} privileges for the {ml-features} in the appropriate spaces.
-Alternatively, `read` {kib} privileges for the {ml-features} and `all` {kib}
-privileges for the index pattern management feature
-* [ ] `manage_pipeline` cluster privilege
-* [ ] `create` and `create_index` index privileges for destination indices
+* [ ] `all` {kib} privileges for the {ml-app} or Discover features in
+the appropriate spaces. Alternatively, `read` {kib} privileges for the {ml-app}  feature and `all` {kib} privileges for the {ipm-app} feature
+* [ ] `manage_pipeline` or `manage_ingest_pipelines` cluster privileges
+* [ ] `create`, `create_index`, and `read` index privileges for destination
```

Comment:
'manage' is also needed for the index already exists check that happens when typing in the index name in the UI.

Reply:
Thanks!  I've added it in 14a09fe4000ab9effcdd5db3f7f65562df278103